cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -757,15 +758,18 @@ def _deserialize_gate_op(
             op = gate(*qubits)
         elif which_gate_type == 'noisechannel':
             which_channel_type = operation_proto.noisechannel.WhichOneof('channel_value')
+            print('dingdong')
```

Comment:
Please clean up debug prints here and below.

Reply:
Removed.